src/search.cpp

```diff
@@ -39,6 +39,7 @@
 #include <unicode/locid.h>
 
 #define MAX_MATCHES_TO_SORT 10000
+#define ANCHOR_TERM "0posanchor "
```

Comment:
It would be better to define `ANCHOR_TERM` only once.

Reply:
In your opinion, which is the best place to define this so that we have access to it in both files? is there any common file where we define constants?

Reply:
There is no common header for the constant. (There is one for the type, but we should not use it).

You can create a new header "constant.h" with this only one `ANCHOR_TERM` for now. We will add more constants later if needed.